### ISSUE TITLE
Force logout and settings got reset

### ISSUE THREAD

**Issue by aozaki-kuro** [BandarHL/BHTwitter#218]
As have mentioned in https://github.com/BandarHL/BHTwitter/issues/67 , the problem still exists on my side.

Device: iPhone 15 Pro
iOS version: iOS 17.3 (21D5044a)
App version: https://github.com/BandarHL/BHTwitter/releases/tag/4.2-1

The App will perform a force logout along with the total reset of BHTwitter settings and the App settings (e.g. Media auto playback got enabled again).
The problem seemed to have existed for a long time and I cannot find out the reason why this happens (I have found no logs for this).

**Comment by DEREZANO** [comment]
@aozaki-kuro hey did you manage to find a solution?

**Comment by aozaki-kuro** [comment]
@DEREZANO Yes, it turned out that I used the dev cert instead of production cert. After switching to production cert, the settings no longer get reset.

**Comment by DEREZANO** [comment]
@aozaki-kuro im sorry but how do i find that the prod cert?
